Sources/ApplicativeRouter/Combinators.swift

```diff
@@ -44,20 +45,27 @@ public func pathParam<A>(_ f: PartialIso<String, A>) -> Router<A> {
 public func queryParam<A>(_ key: String, _ f: PartialIso<String?, A>) -> Router<A> {
   return .init(
     parse: { route in
-      return f.apply(route.query[key]).map { (route, $0) }
+      return f.apply(route.params[key]).map { (route, $0) }
```

Comment:
You'll see, later, that `params` is a computed property converting the `String?` query into `[String: String]`. We can definitely optimize here when need be.